processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -148,6 +134,11 @@ public GroupByMergingQueryRunnerV2(
         String.format("druid-groupBy-%s_%s", UUID.randomUUID(), query.getId())
     );
 
+    final LimitedTemporaryStorage temporaryStorage = new LimitedTemporaryStorage(
```

Comment:
That's true, but it does encapsulate resources if it's ever used at all, and in that case a it does need to be closed. Scoping it to the Sequence with the other stuff is a straight forward way of doing that. Otherwise we'd need to have some other way of making sure it gets closed if used.